backend/api/serializers/ExclusionReport.py

```diff
@@ -119,16 +119,27 @@ class ExclusionReportUpdateSerializer(serializers.ModelSerializer):
     """
     Update Serializer for the Compliance Report
     """
-    status = ComplianceReportWorkflowStateSerializer(required=False)
-    type = SlugRelatedField(slug_field='the_type', read_only=True)
+    actions = serializers.SerializerMethodField()
```

Comment:
Do you think there's any need/desire to have `actor` here, too, as with compliance reporting?

Reply:
we can sure